modules/install-osp-deploy-dualstack.adoc

```diff
@@ -164,4 +164,11 @@ ipv6.addr-gen-mode=0
 ----
 
 After you create and edit the file, reboot the installation host.
+====
+
+[NOTE]
+====
+The `ip=dhcp,dhcp6` kernel argument, which is set to all the nodes, generates a Network Manager connection profile that is activated on multiple interfaces simultaneously.
+This means that any additional network would have the same connection enforced with the same UUID. If an interface specific configuration is desired, a new connection profile
+should be created to the needed interface, which will make the interface to be pulled out of the default connection.
```

Comment:
```suggestion
```suggestion
Because of this behavior, any additional network will have the same connection enforced with an identical UUID. If you need an interface-specific configuration, create a new connection profile for that interface so that the interface is pulled out of the default connection. 
```
Not 100% on the meaning here--what are other terms you might use in place of "pulled out"?

Reply:
stop having the default connection being enforced on it